packages/msi/msi/Product.wxs

```diff
@@ -6,13 +6,23 @@
 
 <?define UpgradeCodeStandard = "b0fee933-ccd2-467c-8fe4-bb0ac6a099c8" ?>
 <?define UpgradeCodeEnterprise = "4ABDD5C7-E1E1-41A6-8119-DCE65634A6CC" ?>
```

Comment:
Not required as this PR but we should remove the enterprise package since that is something we do not do anymore. Essentially we have the same package, just for amd64 and arm64. From what I see in the build pipeline we do not even build enterprise anymore.

Reply:
just a not on this - we need to at least keep the `UpgradeCodeEnterprise` and `OtherSKU` logic, as that protects us from installing the standard package side-by-side with the enterprise one

Reply:
Great idea.